app/addons/replication/components/common-table.js

```diff
@@ -381,7 +381,7 @@ export class ReplicationTable extends React.Component {
   render () {
 
     return (
-      <Table striped>
+      <Table striped style={{marginBottom: '100px'}}>
         <thead>
           <tr>
             <th className="replication__table-bulk-select">
```

Comment:
Unrelated to your changes... but I noticed on lines 403 and 405, it should be `startTime` and not `statusTime` so the sorting matches the values displayed for that column.